### ISSUE TITLE
Error: reference to packed field is unaligned

### ISSUE THREAD

**Issue by l2dy** [benfred/remoteprocess#71]
Could not build remoteprocess 0.4.11 with rustc 1.69.0. There are several [E0793](https://doc.rust-lang.org/beta/error_codes/E0793.html) errors.

```
error[E0793]: reference to packed field is unaligned
   --> ~/.cargo/registry/src/github.com-1ecc6299db9ec823/remoteprocess-0.4.11/src/freebsd/kinfo_proc.rs:303:40
    |                                                                                                                                                                                                 
303 |         unsafe { ::std::mem::transmute(self._bitfield_1.get(0usize, 16u8) as u64) }
    |                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: fields of packed structs are not properly aligned, and creating a misaligned reference is undefined behavior (even if that reference is never dereferenced)
    = help: copy the field contents to a local variable, or replace the reference with a raw pointer and use `read_unaligned`/`write_unaligned` (loads and stores via `*p` must be properly aligned even when using raw pointers)
```

**Comment by benfred** [comment]
Thanks for the heads up! This should be fixed by #74  (which also adds freebsd testing to GHA)

**Comment by benfred** [comment]
fix is in v0.4.12 https://crates.io/crates/remoteprocess
